### PR TITLE
Add Cosmos testnet to Travis

### PR DESCRIPTION
**Pull request by willclarktech**
Part of #1220

### CODE REVIEW

File: scripts/cosmos/.gaiad/config/addrbook.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "key": "84524d4c173831437f628f5d",
-  "addrs": []
-}
+	"key": "84524d4c173831437f628f5d",
+	"addrs": []
+}
```
Comment (webmaster128) [inline scripts/cosmos/.gaiad/config/addrbook.json]: Why does this use tabs and missing trailing newline now?

Comment (willclarktech) [reply scripts/cosmos/.gaiad/config/addrbook.json]: I didn't touch this myself so I think the gaia application must have done this.

Comment (willclarktech) [reply scripts/cosmos/.gaiad/config/addrbook.json]: Confirmed: starting the application reformats this file.

File: scripts/travis/mode_tests-default.sh [hunk 2]
```diff
@@ -41,6 +41,13 @@ if command -v docker > /dev/null ; then
   export LISK_ENABLED=1
   fold_end
 
+  fold_start "cosmos-start"
+  mkdir -p ./scripts/cosmos/.gaiad/data
+  cp ./scripts/cosmos/priv_validator_state.template.json ./scripts/cosmos/.gaiad/data/priv_validator_state.json
```
Comment (webmaster128) [inline scripts/travis/mode_tests-default.sh]: Can't we move those commands into `./scripts/cosmos/start.sh`? This looks very inconvenient to start manually

Comment (willclarktech) [reply scripts/travis/mode_tests-default.sh]: Overwriting the `priv_validator_state.json` file will effectively reset the blockchain.

Comment (webmaster128) [reply scripts/travis/mode_tests-default.sh]: Which is what the other start scripts do as well. No other `./stop.sh && ./start.sh` would pickup the previous blockchain state

Comment (willclarktech) [reply scripts/travis/mode_tests-default.sh]: Good point.

File: scripts/travis/mode_tests-default.sh [hunk 3]
```diff
@@ -41,6 +41,12 @@ if command -v docker > /dev/null ; then
   export LISK_ENABLED=1
   fold_end
 
+  fold_start "cosmos-start"
+  ./scripts/cosmos/clean.sh
```
Comment (webmaster128) [inline scripts/travis/mode_tests-default.sh]: Do we need this call/script? This is done in start.sh automatically now, right?

Comment (willclarktech) [reply scripts/travis/mode_tests-default.sh]: Ah sorry, thought I'd deleted it.

**Pull request merged**